src/Button/Button.less

```diff
@@ -22,7 +22,7 @@
 
 a, button {
   &.Button {
-    .type_medium;
+    .text--medium;
```

Comment:
From my investigation there are very few references so far to .type_size mixins, the only repo that I found that used them was schools-dashboard so far and only in LESS.